pubsub/setup.py

```diff
@@ -22,7 +22,7 @@
 
 name = "google-cloud-pubsub"
 description = "Google Cloud Pub/Sub API client library"
-version = "1.0.2"
+version = "1.0.3"
```

Comment:
Just a thought - since we have a new feature (the `publisher.stop()` method), shouldn't that be a _minor_ version bump, i.e. `1.1.0`?

Reply:
Good point, will bump the minor.